pkg/retry/config.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	AppID            string        `env:"GITHUB_APP_ID,required"`
 	BigQueryID       string        `env:"BIG_QUERY_ID,required"`
 	BucketURL        string        `env:"BUCKET_URL,required"`
+	DatasetID        string        `env:"DATASET_ID,required"`
```

Comment:
Realizing ill have to pass each table name as well since they are also configurable.